includes/footer.php

```diff
@@ -86,31 +86,23 @@
 
       </div>
       <div class="col-sm-6 col-lg-3 offset-lg-1 mb-3 d-print-none">
-        <h5>Getting Started</h5>
+        <h5>About sanger-tol</h5>
```

Comment:
We already another column called  **About sanger-tol**, probably leave as it is or merge them together.

Reply:
I've renamed things